components/renderer/src/index.js

```diff
@@ -42,15 +39,26 @@ app.get("/api/render", async (req, res) => {
             }
         });
         console.log(`URL ${url}: PDF created`);
-        await browser.close();
```

Comment:
Do we still call `browser.close` somewhere? Or does `webPage.close` also close the browser recursively?